girder/stemserver_plugin/models/stemimage.py

```diff
@@ -1,24 +1,30 @@
-import os
-
 import h5py
 import numpy as np
 
 from girder.api.rest import setResponseHeader, RestException
-from girder.constants import AccessType
+from girder.constants import AccessType, AssetstoreType
+from girder.models.assetstore import Assetstore as AssetstoreModel
 from girder.models.file import File as FileModel
+from girder.models.folder import Folder as FolderModel
+from girder.models.item import Item as ItemModel
 from girder.models.model_base import AccessControlledModel
+from girder.utility.filesystem_assetstore_adapter import (
+    FilesystemAssetstoreAdapter
+)
 
 
 class StemImage(AccessControlledModel):
 
+    IMPORT_FOLDER = 'stem_images'
+    IMPORT_ITEM_NAME = '_imported_images'
+
     def __init__(self):
         super(StemImage, self).__init__()
 
     def initialize(self):
         self.name = 'stemimages'
-
-        self.exposeFields(level=AccessType.READ, fields=(
-           '_id', 'filePath', 'fileId'))
+        self.ensureIndices(('_id', 'fileId'))
```

Comment:
`_id` will have a index by default ( I know this might be cut and paste )

Reply:
Okay. I will get rid of that one.